contributr/contriblog/templates/contriblog/detail.html

```diff
@@ -1,14 +1,14 @@
 {% load markdown_deux_tags %}
 <html>
-<head><title>Contriblog</title></head>
+<head><title>ContriBlog</title></head>
 <body>
```

Comment:
Probably use some kind of inherit here from a `base.html`? We can do this now or later.